app/server/lib/ExportXLSX.ts

```diff
@@ -1,34 +1,106 @@
 import {ActiveDoc} from 'app/server/lib/ActiveDoc';
 import {createExcelFormatter} from 'app/server/lib/ExcelFormatter';
-import {ExportData, exportDoc} from 'app/server/lib/Export';
+import {ExportData, exportDoc, ExportParameters, exportSection, exportTable, Filter} from 'app/server/lib/Export';
 import {Alignment, Border, Fill, Workbook} from 'exceljs';
 import * as express from 'express';
 import log from 'app/server/lib/log';
 import contentDisposition from 'content-disposition';
+import { ApiError } from 'app/common/ApiError';
 
-export interface DownloadXLSXOptions {
+export interface DownloadXLSXOptions extends ExportParameters {
   filename: string;
 }
 
+export function buildDownloadXlsxOptions({
```

Comment:
The `/xlsx` endpoint was called without arguments. Which is not the case for the `/csv` endpoint. I'll try to factorise this, but I'm not sure what is needed. Should I remove the `tableId` option for xlsx export ?

Reply:
We need an helper function because the `/xlsx` endpoint was not sending any parameters so to have an homogeneous interface we decide to have a parameters factory.

`/csv` endpoint take quite some parameters contrary to `/xlsx`.